packages/hadron-build/lib/signtool.js

```diff
@@ -1,12 +1,50 @@
 const debug = require('debug')('hadron-build:signtool');
 const { execFileSync } = require('child_process');
 const path = require('path');
+const fs = require('fs/promises');
+
+const SIGNTOOL_PATH = path.resolve(__dirname, '..', 'signtool/signtool.exe');
+
+async function buildSignTool() {
+  try {
+    await fs.access(SIGNTOOL_PATH);
+  } catch (err) {
+    debug('Signtool not found, building it');
+    const buildScriptDir = path.resolve(__dirname, '..', 'signtool');
+    debug(`Building signtool.exe in ${buildScriptDir}`);
+    console.info(execFileSync('./build.sh', {
+      cwd: buildScriptDir,
+    }).toString());
+  }
+}
+
+async function patchWinInstaller() {
+  // Replaces signtool.exe in electron-winstaller with the emulated version
+  // that uses the notary service.
+  const originalSigntool = require.resolve('electron-winstaller/vendor/signtool.exe');
+  const originalSigntoolBackup = originalSigntool + '.bkp';
+
+  try {
+    await fs.access(originalSigntoolBackup);
+  } catch (err) {
+    debug('Original signtool.exe to be replaced not found');
+    return;
```

Comment:
Isn't this always reaching this line? On a fresh install `signtool.bkp` will not be there.

Made me think, what happens if we forget to replace `signtool.exe`? Whatever route with take here, let's make sure that we cannot end up with unsigned binaries. Perhaps now that you found out about git-attributes we could go back to push the pre-build version?